src/core/base.ts

```diff
@@ -8,7 +11,7 @@ export function Component<T extends Function>(selector: string, options: Element
         } else {
             target.constructor.prototype[observedAttributes] = [];
             target.prototype[bindings].forEach((prop: string) => {
```

Comment:
Are we going to get an error if component has no binding properties?

Reply:
Fixed